ui/app/AppLayouts/Wallet/views/CollectiblesView.qml

```diff
@@ -6,6 +6,10 @@ import StatusQ.Core.Theme 0.1
 import StatusQ.Components 0.1
 import shared.panels 1.0
 
+import utils 1.0
+
+import utils 1.0
```

Comment:
Double import? Nice ;)

Reply:
we're cherry picking commits from master to fix community edit settings, obviously this is because of this. anyway, removed now